apps/Gruntfile.js

```diff
@@ -125,6 +127,12 @@ module.exports = function (grunt) {
         },
         {
           expand: true,
+          cwd: piskelRoot,
+          src: '**',
+          dest: 'build/package/js/piskel/'
```

Comment:
I'll need to replace this copy step with the webpack equivalent once @pcardune's refactor goes in.